src/web/topic/components/CriteriaTable/CriteriaTable.tsx

```diff
@@ -183,7 +187,7 @@ const buildTableDefs = (
  *   b. subsequent rows turn into row objects (`rowData`) that know their row header details (for filtering)
  */
 export const CriteriaTable = () => {
-  const [useSolutionsForColumns, setUseSolutionsForColumns] = useState<boolean>(true);
+  const useSolutionsForColumns = useUseSolutionsForColumns();
```

Comment:
This name is a bit awkward. I think we could actually use this simpler name because the table might include other node types besides solutions in the future:
```suggestion
  const transposed = useTransposed(); // no transpose means solutions are put into the columns
```